docs(cmd): updating cmd documentation

ramfox opened the pull request:
While writing the [RFC CLI commands](qri-io/rfcs#14) document, I figured I would go through and update the documentation on the CLI commands, if any are outdated.

b5 commented on cmd/registry.go:
```diff
@@ -44,36 +40,44 @@ $ qri config set registry.location ""`,
+of your dataset on the registry, and if your dataset is less than 20mbs, 
+Qri will back your dataset up onto the registry.
+
+Published datasets can be found by other peers using the ` + "`qri search`" + ` command.
```
single quotes

b5 commented on cmd/render.go:
```diff
@@ -15,7 +15,15 @@ func NewRenderCommand(f Factory, ioStreams IOStreams) *cobra.Command {
+to render visualizations from your dataset. These visualizations can be charts, 
+graphs, or just display your dataset in a different format.
+
+Use the ` + "`--output`" + ` flag to save the rendered html to a file.
```
single quotes

b5 commented on cmd/render.go:
```diff
@@ -15,7 +15,15 @@ func NewRenderCommand(f Factory, ioStreams IOStreams) *cobra.Command {
+
+Use the ` + "`--output`" + ` flag to save the rendered html to a file.
+
+Use the ` + "`--template`" + ` flag to use a custom template. If no template is
```
single quotes

b5 commented on cmd/save.go:
```diff
@@ -24,20 +24,24 @@ func NewSaveCommand(f Factory, ioStreams IOStreams) *cobra.Command {
+Qri will automatically generate one for you.
+
+When you make an update and save a dataset that you originally added from a different
+peer, the dataset gets renamed from ` + "`peers_name/dataset_name`" + ` to ` + "`my_name/dataset_name`" + `.
```
single quotes

b5 commented on cmd/setup.go:
```diff
@@ -21,17 +21,20 @@ func NewSetupCommand(f Factory, ioStreams IOStreams) *cobra.Command {
+running setup. If setup has already been run, by default Qri won’t let you 
+overwrite this info.
+
+Use the ` + "`--remove`" + ` to remove your Qri repo. This deletes your entire repo, 
```
single quotes

b5 commented on cmd/validate.go:
```diff
@@ -48,20 +48,26 @@ schema for dataset "me/foo"
 Using validate this way is a great way to see how changes to data or schema
 will affect a dataset before saving changes to a dataset.
 
+You can get the current schema of a dataset by running the ` + "`qri get structure.schema`" + `
```
single quotes

ramfox commented:
Okay before I go and remove all the back ticks, the reason I changed it in the first place is if we auto generate docs from this using cobra, then we won't have to go through and change all the single quotes to back ticks for the markdown files.

b5 commented:
Ooooooh ok fantastic. Def don't remove then. I think it'd be great if we had a `make cli-docs` command that generated the docs in question. We should also add a `Package cmd` note in `cmd/cmd.go` that explains why the backticks so others don't yell at us later :)

ramfox commented:
@b5 
Added package notes, and added a `docs.go` file and script to the Makefile

a lil bit down and dirty, but didn't want to dump too much time into it.
